Makefile

```diff
@@ -293,7 +302,6 @@ _img_build: $(DIR)/.common
 $(DIR)/.common: $(COMMON_DIR)
 	$(RM) -r $(DIR)/.common
 	cp -r $(COMMON_DIR) $(DIR)/.common
-
```

Comment:
You've deleted an existing blank between targets, I think this will hurt readability.

Reply:
fixed, thanks!